internal/stores/hostdb.go

```diff
@@ -51,6 +53,13 @@ type (
 		Type      string
 	}
 
+	// dbBlocklistEntry defines a table that stores the host blocklist.
+	dbBlocklistEntry struct {
+		Model
+		Entry string   `gorm:"unique;index;NOT NULL"`
```

Comment:
Do we need a type field here? The type would specify what kind of rule this is and the entry would specify the value. e.g.

"netaddress" and "foo.com"
"hostkey" and "somekey"

Reply:
Maybe, I thought about it as well because we'll probably going to need it if we are going to add support for subnet masks and stuff like that. I'm still looking for a better way to do that. For the time being it just checks the "host" portion of the `NetAddress` and this works for our default blocklist so I haven't added a type yet.